variables.tf

```diff
@@ -45,6 +45,12 @@ variable "max_retries" {
   description = "The maximum number of times to retry a failing job"
 }
 
+variable "timeout" {
+  type        = number
+  default     = null
```

Comment:
shouldn't this value be 2880 instead of null ?

Reply:
in this case i chose for null to leave it default  when you don't define the variable.
 Because with glue streaming it requires other value

Reply:
Ah.. ok got it.. I was confused with the description field..